src/components/view-more/view-more.js

```diff
@@ -0,0 +1,62 @@
+function viewMore() {
+	if (document.querySelector(".js-view-more-panel")) {
+		const viewMores = document.querySelectorAll(".js-view-more-panel");
+		const mobileBreakpoint = 499;
+
+		viewMores.forEach((viewMore)=> {
+			let showMoreCTAs = viewMore.querySelectorAll(".js-view-more-panel-cta");
+			let isMobileOnly = viewMore.classList.contains("js-view-more-panel--mobile-only") ? true : false;
+			let windowWidth = window.innerWidth;
+
+			const checkTabpanelHeight = (tabpanelToCheck) => {
+				let tabpanelToCheckInner = tabpanelToCheck.querySelector(".js-view-more-panel-body");
+				if (tabpanelToCheckInner.clientHeight < tabpanelToCheckInner.scrollHeight) {
+					tabpanelToCheck.classList.add("view-more-panel--cropped");
+				} else {
+					tabpanelToCheck.classList.remove("view-more-panel--cropped");
+				}
+			};
+			if (!isMobileOnly || (windowWidth <= mobileBreakpoint)) {
+				checkTabpanelHeight(viewMore);
+			}
+			
+			const expandTabpanelText = (tabToExpand, CTAclicked) => {
+				CTAclicked.querySelector(".js-view-more-panel-cta-text").innerHTML = "Less";
+				tabToExpand.classList.remove("view-more-panel--cropped");
+				tabToExpand.classList.add("view-more-panel--expanded");
+			};
+	
+			const contractTabpanelText  = (tabToContract, CTAclicked) => {
+				window.scrollTo({
+					behavior: "smooth",
+					top: viewMore.getBoundingClientRect().top - document.body.getBoundingClientRect().top - 80,
```

Comment:
where does `80` come from? seems like a number to pull out into a variable with a name?

Reply:
Well when that thing scrolls up to the top that's a bit of a buffer to clear the nav (in some cases) but just kind of a buffer so the user can get their bearings.

Reply:
yah, i just meant we should avoid a 'magic number': https://en.wikipedia.org/wiki/Magic_number_(programming)